<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1309</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>1281</width>
      <height>581</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="title">
     <string>Quản lí nhân viên</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QPushButton" name="btnAdd">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>530</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(7, 28, 42);
border: 2px solid rgb(7, 144, 245);   /* Blue border */
border-radius: 8px;
color:white;</string>
     </property>
     <property name="text">
      <string>Thêm</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnEdit">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>530</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(7, 28, 42);
border: 2px solid rgb(7, 144, 245);   /* Blue border */
border-radius: 8px;
color:white;</string>
     </property>
     <property name="text">
      <string>Sửa</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDel">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>530</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(7, 28, 42);
border: 2px solid rgb(7, 144, 245);   /* Blue border */
border-radius: 8px;
color:white;</string>
     </property>
     <property name="text">
      <string>Xóa</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Mã nhân viên</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="maNV">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>110</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(7, 28, 42);
border: 2px solid rgb(7, 144, 245);   /* Blue border */
border-radius: 8px;
color:white;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tenNV">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>150</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(7, 28, 42);
border: 2px solid rgb(7, 144, 245);   /* Blue border */
border-radius: 8px;
color:white;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Tên nhân viên</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>310</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Chức vụ</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="password">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>270</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(7, 28, 42);
border: 2px solid rgb(7, 144, 245);   /* Blue border */
border-radius: 8px;
color:white;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>270</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Mật khẩu</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnReset">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>530</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(7, 28, 42);
border: 2px solid rgb(7, 144, 245);   /* Blue border */
border-radius: 8px;
color:white;</string>
     </property>
     <property name="text">
      <string>Làm mới</string>
     </property>
    </widget>
    <widget class="QPushButton" name="searchbutton">
     <property name="geometry">
      <rect>
       <x>943</x>
       <y>60</y>
       <width>121</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(7, 28, 42);
border: 2px solid rgb(7, 144, 245);   /* Blue border */
border-radius: 8px;
color:white;</string>
     </property>
     <property name="text">
      <string>Tìm kiếm</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="search">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>60</y>
       <width>521</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(7, 28, 42);
border: 2px solid rgb(7, 144, 245);   /* Blue border */
border-radius: 8px;
color:white;</string>
     </property>
     <property name="placeholderText">
      <string>Nhập tên hoặc số điện thoại</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableNV">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>100</y>
       <width>861</width>
       <height>421</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>661</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableWidget {
    background-color: rgb(7, 28, 42);     /* cell background */
    color: white;                  /* text color */
    gridline-color: #dcdcdc;         /* grid line color */
    font-size: 14px;
}

QTableWidget::item:selected {
    background-color: #85c1e9;       /* selected row background */
    color: white;                    /* selected text color */
}
QHeaderView::section {
    background-color: rgb(25, 47, 63);
    color: white;
    font-weight: bold;
}

QScrollBar:horizontal {
    background: rgb(0, 0, 0);
    height: 10px;
    margin: 0px;
}

QScrollBar::handle:horizontal {
    background: #3498db;
    min-width: 20px;
    border-radius: 5px;
}

QScrollBar::add-line:horizontal,
QScrollBar::sub-line:horizontal {
    background: none;
    width: 0px;
}
</string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideRight</enum>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <property name="gridStyle">
      <enum>Qt::NoPen</enum>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>150</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Mã nhân viên</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Tên nhân viên</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Số điện thoại</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Email</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Mật khẩu</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Chức vụ</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ngày vào làm</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Trạng thái</string>
      </property>
     </column>
    </widget>
    <widget class="QLineEdit" name="email">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>230</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(7, 28, 42);
border: 2px solid rgb(7, 144, 245);   /* Blue border */
border-radius: 8px;
color:white;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>230</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>350</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ngày đăng ký</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateDK">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>350</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(7, 28, 42);
border: 2px solid rgb(7, 144, 245);   /* Blue border */
border-radius: 8px;
color:white;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="role">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>310</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(7, 28, 42);
border: 2px solid rgb(7, 144, 245);   /* Blue border */
border-radius: 8px;
color:white;</string>
     </property>
     <item>
      <property name="text">
       <string>STAFF</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ADMIN</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="phone">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>190</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(7, 28, 42);
border: 2px solid rgb(7, 144, 245);   /* Blue border */
border-radius: 8px;
color:white;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Số điện thoại</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>390</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Tình Trạng</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>390</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Hoạt động</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnDangXuat">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>10</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(7, 28, 42);
border: 2px solid rgb(7, 144, 245);   /* Blue border */
border-radius: 8px;
color:white;</string>
    </property>
    <property name="text">
     <string>Đăng Xuất</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1311</width>
      <height>701</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(18, 38, 52);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btnNV_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>111</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(7, 28, 42);
border: 2px solid rgb(7, 144, 245);   /* Blue border */
border-radius: 8px;
color:white;</string>
     </property>
     <property name="text">
      <string>Quản lí nhân viên</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnHD">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>121</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(7, 28, 42);
border: 2px solid rgb(7, 144, 245);   /* Blue border */
border-radius: 8px;
color:white;</string>
     </property>
     <property name="text">
      <string>Quản lí hóa đơn</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDHN">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>10</y>
       <width>131</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(7, 28, 42);
border: 2px solid rgb(7, 144, 245);   /* Blue border */
border-radius: 8px;
color:white;</string>
     </property>
     <property name="text">
      <string>Quản lí đồng hồ nước</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnKH">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>121</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(7, 28, 42);
border: 2px solid rgb(7, 144, 245);   /* Blue border */
border-radius: 8px;
color:white;</string>
     </property>
     <property name="text">
      <string>Quản lí khách hàng</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>371</width>
       <height>451</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(20, 43, 59);
 border: 2px solid rgb(1, 52, 88);      /* Blue outline */
    border-radius: 15px;            /* Rounded corners */</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
   <zorder>frame</zorder>
   <zorder>groupBox</zorder>
   <zorder>btnDangXuat</zorder>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>search</sender>
   <signal>returnPressed()</signal>
   <receiver>searchbutton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>680</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>1013</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tenNV</sender>
   <signal>returnPressed()</signal>
   <receiver>email</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>email</sender>
   <signal>returnPressed()</signal>
   <receiver>password</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
